source/client/output_formatter_impl.cc

```diff
@@ -157,6 +157,100 @@ YamlOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) co
+                                                      : fmt::format("{}", statistic.raw_min());
+        const std::string s_max = statistic.has_max() ? formatProtoDuration(statistic.max())
+                                                      : fmt::format("{}", statistic.raw_max());
+        const std::string s_mean = statistic.has_mean() ? formatProtoDuration(statistic.mean())
+                                                        : fmt::format("{}", statistic.raw_mean());
+        const std::string s_pstdev = statistic.has_pstdev()
+                                         ? formatProtoDuration(statistic.pstdev())
+                                         : fmt::format("{}", statistic.raw_pstdev());
+
+        ss << fmt::format("{} ({} samples)", statIdtoFriendlyStatName(statistic.id()),
+                          statistic.count())
+           << std::endl;
+
+        // Descriptive statistics
+        ss << "Min,Mean,Max,Pstdev" << std::endl;
+        ss << fmt::format("{},{},{},{}", s_min, s_mean, s_max, s_pstdev) << std::endl;
+
+        bool header_written = false;
+        iteratePercentiles(statistic, [&ss, this, &header_written](
+                                          const nighthawk::client::Percentile& percentile) {
+          const auto p = percentile.percentile();
+          // Don't show the min / max, as we already show that above.
+          if (p > 0 && p < 1) {
+            // Table headers
+            if (!header_written) {
+              ss << "Percentile,Count,Value(microseconds),Value" << std::endl;
+              header_written = true;
+            }
+            auto s_percentile = fmt::format("{:.{}g}", p, 8);
+            ss << fmt::format("{},{},{},{}", s_percentile, percentile.count(),
+                              Envoy::Protobuf::util::TimeUtil::DurationToMicroseconds(percentile.duration()),
+                              percentile.has_duration()
+                                  ? formatProtoDuration(percentile.duration())
+                                  : fmt::format("{}", static_cast<int64_t>(percentile.raw_value())),
+                              15)
+               << std::endl;
+          }
+        });
+        ss << std::endl;
+      }
+      
+      // Counters
+      ss << fmt::format("{},{},{}", "Counter", "Value", "Per second") << std::endl;
+      for (const auto& counter : result.counters()) {
+        const auto nanos =
+            Envoy::Protobuf::util::TimeUtil::DurationToNanoseconds(result.execution_duration());
+        ss << fmt::format("{},{},{:.{}f}", counter.name(), counter.value(),
+                          counter.value() / (nanos / 1e9), 2)
+           << std::endl;
+      }
+      ss << std::endl;
+    }
+  }
+  return ss.str();
+}
+
+std::string CsvOutputFormatterImpl::formatProtoDuration(
+    const Envoy::ProtobufWkt::Duration& duration) const {
+  auto c = Envoy::Protobuf::util::TimeUtil::DurationToMicroseconds(duration);
```

Comment:
Can we use a longer, more descriptive name instead of just `c`, it isn't immediately obvious what `c` stands for.

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Sure! You are totally right.